### PR TITLE
Fixes #13615: Bad explanation for generic variable overide priority cause people making the opposite of what is needed

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/13615

### CODE REVIEW

File: rudder-web/src/main/scala/com/normation/rudder/web/components/DirectiveEditForm.scala [hunk 1]
```diff
@@ -420,8 +420,11 @@ class DirectiveEditForm(
             <span tooltipid="priorityId" class="ruddericon tooltipable glyphicon glyphicon-question-sign" title=""></span>
             <div class="tooltipContent" id="priorityId">
               <h4> Priority </h4>
-              <p>Priority determines which <b> unique </b> Directive will be applied.</p>
-              <p>Unique Directives can be applied only once (ie. Time Settings), so only the highest priority will be appllied.</p>
+              <p>Priority has two uses depending if the technique from which that directive is derived is <b>Unique</b> or not.</p>
+              <p>Unique directives can be applied only once (for example Time Settings), so only the highest priority will be applied.</p>
+              <p>For <b>non-unique</b> directives, priority is used to choose the order of the directive linearisation when applicable. The
```
Comment (amousset) [inline rudder-web/src/main/scala/com/normation/rudder/web/components/DirectiveEditForm.scala]: I don't think linearisation is clear for users. Maybe application?

**Comment by fanf** [comment]
PR rebased

File: rudder-web/src/main/scala/com/normation/rudder/web/components/DirectiveEditForm.scala [hunk 2]
```diff
@@ -420,8 +420,11 @@ class DirectiveEditForm(
             <span tooltipid="priorityId" class="ruddericon tooltipable glyphicon glyphicon-question-sign" title=""></span>
             <div class="tooltipContent" id="priorityId">
               <h4> Priority </h4>
-              <p>Priority determines which <b> unique </b> Directive will be applied.</p>
-              <p>Unique Directives can be applied only once (ie. Time Settings), so only the highest priority will be appllied.</p>
+              <p>Priority has two uses depending if the technique from which that directive is derived is <b>Unique</b> or not.</p>
+              <p>Unique directives can be applied only once (for example Time Settings), so only the highest priority will be applied.</p>
+              <p>For <b>non-unique</b> directives, priority is used to choose the order of the directive application when applicable. The
+                highest priority directive comes first. Note that overriding variables definitions are the ones coming last.</p>
```
Comment (amousset) [inline rudder-web/src/main/scala/com/normation/rudder/web/components/DirectiveEditForm.scala]: > Note that overriding variables definitions are the ones coming last.

.. so the used value will be the one with the lowest priority

**Comment by fanf** [comment]
PR rebased

**Pull request merged**